src/libs/test.ts

```diff
@@ -0,0 +1,37 @@
+counter = 0;
+
+function incrementCounter() {
+  counter++;
+}
+
+function sum(a: number, b: number) {
+  return a + b;
+}
+
+function multiply(a: number, b: number) {
+  return a * b;
+}
+
+const apiKey = process.env.API_KEY;
+
+function fibonacci(n: number): number {
+  if (n <= 1) {
+    return n;
+  }
+  return fibonacci(n - 1) + fibonacci(n - 2);
+}
+
+function myFunction() {
+  console.log("This is my function.");
+}
+
+const myVariable = "This is my variable";
+
+
+const num: number = "some random string";
```

Comment:
@trag-bot Type mismatch. A string is being assigned to a variable explicitly typed as number.